modules/nux/modules.php

```diff
@@ -490,20 +490,22 @@ static public function add($id, $details) {
         self::$services[$id] = $details;
     }
     static public function oauth2_setup($config) {
-        $settings = array();
-        $settings = get_ini($config, 'oauth2.ini', true);
-        if (!empty($settings)) {
-            foreach ($settings as $service => $vals) {
+        $services = ['gmail', 'outlook'];
```

Comment:
This should be dynamic. Take array keys of the new config/oauth2.php

Reply:
Done, I'm using config helpers to access to oauth2.php